site/docs/api-quickstart.md

```diff
@@ -102,13 +102,13 @@ When using the Iceberg API directly, type IDs are required. Conversions from oth
 
 When a table is created, all IDs in the schema are re-assigned to ensure uniqueness.
 
-### Convert a schema from Avro
+### Convert a schema from Apache Avro
 
-To create an Iceberg schema from an existing Avro schema, use converters in `AvroSchemaUtil`:
+To create an Iceberg schema from an existing [Apache Avro schema](https://avro.apache.org/), use converters in `AvroSchemaUtil`:
 
 ```scala
 import org.apache.iceberg.avro.AvroSchemaUtil
-import org.apache.avro.Schema.Parser
+import org.apache.iceberg.shaded.org.apache.avro.Schema.Parser
```

Comment:
This is ugly, but I guess necessary right now. Eventually, I'd like to remove Avro from the public API and move it into an iceberg-avro package. That way we don't have conflicts with shading like this.

Reply:
I'm not pleased with this as well. I'm not sure if we can get rid of the Avro API completely. I can also see that you want to convert the Iceberg schema back to an Avro representation.

Reply:
I've reverted this line until Iceberg 0.8 gets released.